hedera-node/src/main/java/com/hedera/services/state/expiry/MonotonicFullQueueExpiries.java

```diff
@@ -89,4 +89,12 @@ public String toString() {
 					.toString();
 		}
 	}
+
+	Deque<ExpiryEvent> getAllExpiries() {
```

Comment:
Should be `public`

Reply:
This is only used by a unit test, so package-private is the maximum accessibility needed.